paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheetViewModel.kt

```diff
@@ -140,10 +140,6 @@ internal class PaymentSheetViewModel @Inject internal constructor(
         return outputLiveData
     }
 
-    // Holds a reference to the last selected payment method while checking out with Google Pay.
-    // If Google Pay is cancelled or fails, it will be set again as the selected payment method.
-    internal var lastSelectedPaymentMethod: PaymentSelection? = null
```

Comment:
We no longer update `selection` to `PaymentSelection.GooglePay` when the user opens the Google Pay sheet. Therefore, we don’t need this property anymore to keep track of the payment method that was selected before.